integration/package.json

```diff
@@ -278,6 +279,7 @@
   "scripts": {
     "build-wasm": "cd .. && make wasm",
     "install-pkg": "npm run build-wasm && npm install --no-save ../pkg-node",
+    "replay": "npm run install-pkg && node ./replay.js",
```

Comment:
I'm not familiar with what this target does.

Reply:
Oh, oops. This shouldn't have been committed.